src/publisher.js

```diff
@@ -27,11 +27,10 @@ Publisher.prototype.publish = function () {
 	// publish template $pactHost/pacts/provider/$provider/consumer/$client/$version
 	var uris = _.chain(options.pactUrls)
 		.map(function (uri) {
-			uri = path.normalize(uri);
-			if (fs.statSync(uri).isDirectory()) {
-				logger.debug('we are a dir: ' + uri);
+			var localFileOrDir = path.normalize(uri);
+			if (!(/^http/.test(uri)) && fs.statSync(localFileOrDir).isDirectory()) {
```

Comment:
Doesn't fstat return undefined if the part is no good? This would cause an error of it tried to call isDirectory.